test/no-duplicate-modifiers.js

```diff
@@ -0,0 +1,58 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/no-duplicate-modifiers';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const ruleError = {ruleId: 'no-duplicate-modifiers'};
+const header = `const test = require('ava');\n`;
```

Comment:
I still see it when viewing the full diff:

<img width="520" alt="screen shot 2016-08-30 at 20 50 10" src="https://cloud.githubusercontent.com/assets/170270/18102611/69eeb21e-6ef3-11e6-854e-2e5f5dc2946c.png">